js/packages/teams-ai/src/authentication/Authentication.ts

```diff
@@ -245,6 +254,16 @@ export class AuthenticationManager<TState extends TurnState> {
     }
 }
 
+/**
+ * Settings used to configure user authentication through the OAuthPrompt.
+ */
+export type OAuthSettings = OAuthPromptSettings & {
```

Comment:
Please check with Chaoyi & align on how to do casing for OAuth (oAuth, oauth, etc)

Reply:
Ok

Reply:
`OAuth` looks good to me, as it's used in the RFC spec. For variables that need to start with lowercase based on the naming rule, we may use `oAuth` in my opinion. @singhk97 @corinagum